INSTALL.rst.txt

```diff
@@ -135,50 +109,39 @@ Use the command::
 
   git clone https://github.com/scipy/scipy.git
 
-Before building and installing from git, remove the old installation
-(e.g. in /usr/lib/python3.4/site-packages/scipy or
+If you have an older version of Scipy installed and didn't install that with
+pip, you need to manually uninstall it first (remove from e.g. in
+/usr/lib/python3.4/site-packages/scipy or
 $HOME/lib/python3.4/site-packages/scipy).  Then type::
 
   cd scipy
   git clean -xdf
-  python setup.py install
+  pip install . --user
```

Comment:
Hmm, ``pip`` 8.0 contains a bug where it doesn't show the build log even though you ask it to with ``--verbose``. So I'll revert this change.